src/modules/vmware/application/use-cases/get-host-metrics.use-case.spec.ts

```
@@ -0,0 +1,203 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { NotFoundException } from '@nestjs/common';
+import { GetHostMetricsUseCase } from './get-host-metrics.use-case';
+import { IVmwareService } from '../../domain/interfaces/vmware.service.interface';
+import { IServerRepository } from '../../../servers/domain/interfaces/server.repository.interface';
+import { Server } from '../../../servers/domain/entities/server.entity';
```

ESLint

[error] 'Server' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)